### DISCUSSION TITLE
So can github actions and workflows completely kill CI/CD platforms like Jenkins?

### DISCUSSION THREAD

**Discussion by VijayKrishn** [github/india#34] [Satellite India 2021 Q&A] [answered]
If all the automation can be done using github, can we completely eliminate the usage of CI/CD platforms like jenkins?

**Comment by dhruvg20** [comment, marked as answer]
[Migrating from Jenkins to GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/migrating-from-jenkins-to-github-actions) would be able to help your plan on migrating from Jenkins to Actions.  🙇🏽

**Comment by FRTMentor** [comment]
[Migrating from Jenkins to GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/migrating-from-jenkins-to-github-actions) would be able to help your plan on migrating from Jenkins to Actions
